lib/src/main/kotlin/com/lemonappdev/konsist/core/declaration/KoInterface.kt

```diff
@@ -1,14 +1,14 @@
 package com.lemonappdev.konsist.core.declaration
 
 import com.lemonappdev.konsist.core.cache.KoDeclarationCache
-import org.jetbrains.kotlin.lexer.KtTokens
 import org.jetbrains.kotlin.psi.KtClass
+import org.jetbrains.kotlin.psi.psiUtil.hasActualModifier
+import org.jetbrains.kotlin.psi.psiUtil.hasExpectModifier
 
 class KoInterface private constructor(private val ktClass: KtClass) : KoComplexDeclaration(ktClass) {
+    fun hasActualModifier() = ktClass.hasActualModifier()
```

Comment:
Let's use hasModifiers(KoModifier.X) for every modifier property because "kt methods" may have additional behavior